examples/boid_flockers/boid_flockers/boid.py

```diff
@@ -94,11 +95,11 @@ def step(self):
         """
 
         neighbors = self.model.space.get_neighbors(self.pos, self.vision, False)
-        self.velocity += (
+        self.direction_vector += (
             self.cohere(neighbors) * self.cohere_factor
             + self.separate(neighbors) * self.separate_factor
             + self.match_heading(neighbors) * self.match_factor
-        ) / 2
```

Comment:
The `/ 2` affects the result. This is `self.velocity +=`, not `self.velocity =`.

Reply:
Good catch, you're totally right. I will reply in the main thread more extensively.